.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -8,6 +8,6 @@ attention.
 
 * Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
 * Have you followed the [contributor guidelines](https://github.com/elastic/rally/blob/master/CONTRIBUTING.md)?
-* Have you run `make test` and `make it` and both finish without errors?
+* Have you run `make lint`, `make test` and `make it` and they finish without errors?
```

Comment:
Regarding your PR comment we want to ensure all tests pass before raising a community PR is raised so I suggest we replace this with ``Have you run `make check-all` successfully?`` This reduces time spent in the PR review process.

Reply:
Thanks, done.